index.js

```diff
@@ -9,11 +9,12 @@ module.exports = {
     'indent': [2, 2, { SwitchCase: 1 }],
     'key-spacing': [2, { 'beforeColon': false, 'afterColon': true }],
     'linebreak-style': [2, 'unix'],
-    'max-depth': [2, 3],
+    "max-depth": ["error", 5],
```

Comment:
Noooo. Why depth of 5? Also this rule is not following the eslint of this project...